docs/resources/README.md

```diff
@@ -0,0 +1,16 @@
+# Resources
+
+## Resource Detector
+
+OpenTelemetry .NET SDK provides a resource detector for detecting resource
+information from the `OTEL_RESOURCE_ATTRIBUTES` and `OTEL_SERVICE_NAME`
+environment variables.
+
+Custom resource detectors can be implemented:
+
+* ResourceDetectors should inherit from
+  `OpenTelemetry.Resources.IResourceDetector`, (which belongs to the
+  [OpenTelemetry](../../src/OpenTelemetry/README.md) package), and implement
+  the `Detect` method.
+
+A demo ResourceDetector is shown [here](../trace/extending-the-sdk/MyResourceDetector.cs).
```

Comment:
I think it would be better to simply show the code snippet to create a custom `ResourceDetector` here instead of pointing to `docs/trace/..`.

You could then add code snippets that show how to add this custom `ResourceDetector` to all the three signals using both `Sdk.Create/LoggerFactory.Create` and `Extensions.Hosting` approach.

Reply:
Let me know if you would like to address these in this PR itself.